### DISCUSSION TITLE
Production (.org) vs Staging (.net)

### DISCUSSION THREAD

**Discussion by TechShooter** [openfoodfacts/open-prices#408] [Q&A] [answered]
I just noticed that the .org website is the "official" one, right? So what about the .net one? And I suppose there isn't yet a way to move .net prices to .org, right?
I initially thought they were the same, but then I noticed prices and my profile were different

.org: https://prices.openfoodfacts.org/app/
.net: https://prices.openfoodfacts.net/app/

### Related issues

- https://github.com/openfoodfacts/open-prices-frontend/issues/876

**Comment by raphodn** [comment, marked as answer]
Hi @TechShooter the .org is the production environment. And .net is the staging/testing environment.
Similar to https://world.openfoodfacts.org & https://world.openfoodfacts.net

**Reply by raphodn** [reply]
Did you add many prices to the .net environment ?
Via the web app ? Or via API ?

**Reply by TechShooter** [reply]
Yes, I added many prices to the .net environment, via the web app.
I'm not much of a developer, more of a contributer who found the .net site, honestly I think it can be confusing for who doesn't know about testing etc.
Thanks for clarifying though!

**Reply by raphodn** [reply]
Ok that sucks. How many prices ?

I can try to see how that data (prices, but also proofs, locations, & products) could be migrated to prod.

And we could add a top banner in the .net app saying it's a staging environnement...

**Reply by TechShooter** [reply]
151, my username is cloud019998
Thanks!

**Reply by raphodn** [reply]
sorry, i haven't forgotten, just haven't had the time...

**Reply by TechShooter** [reply]
No worries! Thank you
